client/src/util/helpers.ts

```diff
@@ -246,11 +285,15 @@ export default {
     };
   },
 
-  getTranslationOfComplaintType(complaintType: string, i18n: I18n) {
-    const translatedType = hpdComplaintTypeTranslations.get(complaintType);
-    if (!translatedType) {
-      reportError(`The HPD Complaint type "${complaintType}" isn't internationalized`);
-      return complaintType;
-    } else return i18n._(translatedType);
+  translateComplaintType,
+  translateContactTitle,
+  translateContactTitleAndIncludeEnglish(textToTranslate: string, i18n: I18n) {
+    const translation = translateContactTitle(textToTranslate, i18n);
+    if (i18n.language === "en") return translation;
+    else {
+      const textInEnglish = getContactTitleInEnglish(textToTranslate, i18n);
+      const translationSuffix = i18n._(t`("${textInEnglish}" in English)`);
+      return translation + " " + translationSuffix;
+    }
```

Comment:
I wonder if it might be helpful to provide unit tests for this one, in particular to show folks what this is actually doing in practice.  It would require the actual spanish translations to be in-place though, which might be annoying.  Anyways, just a thought... alternatively, adding a comment mentioning what this is doing in practice, along with example output, might be helpful.